test/extern/diten/externVoidFnWithArray.good

```diff
@@ -1 +1 @@
-externVoidFnWithArray.chpl:8: error: illegal use of function that does not return a value: 'voidWithArray'
+externVoidFnWithArray.chpl:9: error: illegal use of function that does not return a value: 'voidWithArray'
```

Comment:
I'm not seeing why this line number would change, though I realize this is one of the tests that's related to my mods w.r.t. passing arrays to externs.

Reply:
I'll add `use CPtr` to this test, so the line number will change.